core/test/unit/middleware/spam-prevention_spec.js

```diff
@@ -19,6 +20,7 @@ describe('Middleware: spamPrevention', function () {
         spyNext = sinon.spy(function (param) {
```

Comment:
improvement request:
when you already work on this test, could you please use the sandbox everyhere?
then for example we don't need to reset `process.hrtime.restore();` in line `97`.